src/REPLMode.jl

```diff
@@ -486,6 +528,50 @@ function do_init!(tokens::Vector{Tuple{Symbol,Vararg{Any}}})
     Pkg3.API.init(pwd())
 end
 
+function do_clone!(env::EnvCache, tokens::Vector{Tuple{Symbol,Vararg{Any}}})
+    isempty(tokens) && cmderror("`clone` take an url to a package to clone")
+    local url
+    basepath = get(ENV, "JULIA_DEV_PATH", default_dev_path())
```

Comment:
So even though this is a "path" to a place, we use `_PATH` and `_PATH` as a suffix in `LOAD_PATH` and `DEPOT_PATH` to indicate a list of places to look for things, whereas this is a single location to put things, which seems more like `Sys.BINDIR` or `JULIA_PKGDIR` so I think this environment variable should be called `JULIA_DEVDIR` or `JULIA_DEV_DIR`. I favor the former since we're [going to have](https://github.com/JuliaLang/julia/issues/20899) a bunch of variables with names like `SRCDIR` and `LIBDIR` so using the name `DEVDIR` seems more consistent.